libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -363,6 +363,28 @@ proc validateAndRelay(g: GossipSub,
                       msgId: MessageId, saltedId: SaltedId,
                       peer: PubSubPeer) {.async.} =
   try:
+    template topic: string = msg.topic
+
+    if msg.data.len > max(512, msgId.len * 10):
+      # If the message is "large enough", let the mesh know that we do not want
```

Comment:
But could a malicious peer flood the network with invalid msgs and thus `idontwant` faster than it gets punished? It might require some testing.